specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1482 @@
+    },
+    "securityDefinitions": {
+        "apim_key": {
+            "type": "apiKey",
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header"
+        }
+    },
+    "x-ms-parameterized-host": {
+        "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
+        "parameters": [
+            {
+                "$ref": "../../Common/ExtendedRegions.json#/parameters/AzureRegion"
+            }
+        ]
+    },
+    "basePath": "/vision/v1.0",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/models": {
+            "get": {
+                "description": "This operation returns the list of domain-specific models that are supported by the Computer Vision API.  Currently, the API only supports one domain-specific model: a celebrity recognizer. A successful response will be returned in JSON.  If the request failed, the response will contain an error code and a message to help understand what went wrong.",
+                "operationId": "List Domain Specific Models",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "schema": {
+                            "$ref": "#/definitions/ListModelsResult"
+                        }
+                    },
+                    "default": {
+                        "schema": {
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful List Domains request": {
+                        "$ref": "./examples//SuccessfulListDomainModels.json"
+                    }
+                }
+            }
+        },
+        "/analyze?overload=url": {
+            "post": {
+                "description": "This operation extracts a rich set of visual features based on the image content. Two input methods are supported -- (1) Uploading an image or (2) specifying an image URL.  Within your request, there is an optional parameter to allow you to choose which features to return.  By default, image categories are returned in the response.",
+                "operationId": "Analyze Image",
```

Comment:
`operationId` must be `groupname_methodname` -- 

`groupname` is used to create a class in most generated languages
`methodname` is used to generate a method name in most languages